identity-service/scripts/fundSolanaFeePayer.js

```diff
@@ -0,0 +1,59 @@
+const solanaWeb3 = require('@solana/web3.js')
+
+// input validation
+if (!process.env.funderPrivateKey || !process.env.feePayerPrivateKey) {
+  console.log('funderPrivateKey and feePayerPrivateKey env vars must be set')
+  console.log('Example usage: `funderPrivateKey=[...] feePayerPrivateKey=[...] node fundSolanaFeePayer.js`')
+  process.exit(1)
+}
+const RPC_ENDPOINT = 'https://api.mainnet-beta.solana.com' // DEVNET is https://api.devnet.solana.com
+// Funder - the private key wallet that will fund the fee payer
+const FUNDER_PRIVATE_KEY = process.env.funderPrivateKey ? JSON.parse(process.env.funderPrivateKey) : []
+// Fee payer - the private key of the wallet exposed in identity to pay for tx's
+const FEE_PAYER_PRIVATE_KEY = process.env.feePayerPrivateKey ? JSON.parse(process.env.feePayerPrivateKey) : []
```

Comment:
do we really need this private key?

Reply:
we don't, we could just pass in the address

Reply:
i think that's better tbh, less private keys around the better

Reply:
agreed, i'll update